src/webview/feedback/app/feedback.tsx

```diff
@@ -0,0 +1,77 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Icon, Stack, Typography } from '@mui/material';
+import { Model } from 'survey-core';
+import { Survey } from 'survey-react-ui';
+import { DefaultProps } from '../../common/propertyTypes';
+import { json } from '../json';
+import { VSCodeMessage } from './vsCodeMessage';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import OpenShiftExtensionIcon from '../../../../images/openshift_icon.png';
+import MicrosoftIcon from '../../../../images/welcome/microsoft.svg';
+import 'survey-core/defaultV2.css';
+import './feedback.css'
+
+export const FeedbackComponent: React.FC<DefaultProps> = ({ }) => {
+
+    const feedbackModal = new Model(json);
+
+    feedbackModal.onComplete.add((sender, options) => {
+        options.showSaveInProgress();
+        VSCodeMessage.postMessage({
+            action: 'postFeedback',
+            data: sender.data
+        });
+        options.showSaveSuccess();
+    });
+
+    feedbackModal.onValueChanged.add((feedbackModal, _question) => {
+        feedbackModal.showNavigationButtons = true;
+    });
+
+    feedbackModal.completeText = 'Submit';
+    feedbackModal.showNavigationButtons = false;
+
+    return (
+        <div className='parentContainer'>
+            <div className='headerContainer'>
+                <header className='header__logo' style={{ width: '50%' }}>
+                    <img className='image__logo' src={OpenShiftExtensionIcon} />
+                    <div style={{ display: 'flex', flexDirection: 'column' }}>
+                        <label style={{ display: 'flex', flexDirection: 'row', textAlign: 'center' }}>
+                            <Typography variant='h4' className='highlight'>OpenShift</Typography>
+                            <Typography variant='h4' style={{ paddingLeft: '1rem' }} className='foreGroundColor'>Toolkit</Typography>
+                        </label>
+                        <Typography variant='caption' style={{ marginTop: '2px' }} className='foreGroundColor'>Your opinion matters to us!</Typography>
```

Comment:
`Your opinion matters to us!` should be written beside the OpenShift logo and in a bigger font size.